cpp/msbuild/ice.proj

```diff
@@ -17,62 +17,11 @@
         <Exec Command="powershell -ExecutionPolicy ByPass -Command &quot;[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12;(New-Object Net.WebClient).DownloadFile('$(NugetURL)', '$(NugetExe)')&quot;"/>
```

Comment:
We can remove this file if we switch to CMake. is a helper project to build all the demos.